events_scrap/events_scrap.py

```diff
@@ -37,7 +39,11 @@ def __init__(self, inventory_url: str, offline_token: str, index: str, es_server
         self.client = ClientFactory.create_client(url=self.inventory_url, offline_token=offline_token)
 
         self.index = index
-        self.es = elasticsearch.Elasticsearch(es_server, http_auth=(es_user, es_pass))
+
+        if es_user:
```

Comment:
```suggestion
        http_auth = None
        if es_user:
```
to make sure we don't have any undefined variable